src/edit_mode/vi/command.rs

```diff
@@ -276,3 +308,20 @@ impl Command {
         }
     }
 }
+
+fn right_bracket_for(c: &char) -> char {
+    match *c {
+        '(' => ')',
+        '[' => ']',
+        '{' => '}',
+        '<' => '>',
+        _ => *c,
+    }
+}
+
+fn is_valid_change_in(c: &char) -> bool {
+    matches!(
+        c,
+        '(' | '[' | '{' | ')' | ']' | '}' | '"' | '\'' | '`' | '<'
```

Comment:
If you support the closing delimiters here, there needs to be the inverse logic for `left_bracket_for` as well, right?

Reply:
Yes, it should support both.. I actually never used the right bracket version. Just tested that it is a thing in neovim.